application/tests/TestCase.php

```diff
@@ -235,4 +240,41 @@ public function warningOn()
 	{
 		error_reporting($this->_error_reporting);
 	}
+	
+	/**
+	*  Is the query a 404
+	* 
+	* @see core/CodeIgniter.php
+	* @param object $RTR    Router object
+	* @param string $class  request controller
+	* @param array  $method request action
+	*/
+	private function _is_404($RTR, $class, $method)
+	{
+		if (empty($class) OR ! file_exists(APPPATH.'controllers/'.$RTR->directory.$class.'.php'))
+		{
+			return TRUE;
+		}
+
+		require_once(APPPATH.'controllers/'.$RTR->directory.$class.'.php');
+		if ( ! class_exists($class, FALSE) OR $method[0] === '_' OR method_exists('CI_Controller', $method))
+		{
+			return TRUE;
+		}
+
+		if (method_exists($class, '_remap'))
+		{
+			$params = array($method, array_slice($URI->rsegments, 2));
```

Comment:
oops! :bomb: I'm so sorry.
I think that is not necessary, so I remove that.
How about you?

https://github.com/kenjis/ci-phpunit-test/commit/cca05807c6ace30aa5933bfd5e7ccc7239237888